ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileClustersPage.tsx

```diff
@@ -17,11 +18,16 @@ function ProfileClustersPage() {
     const [currentDatetime, setCurrentDatetime] = useState<Date>(new Date());
     const pagination = useURLPagination(10);
 
-    const { page, perPage } = pagination;
+    const { page, perPage, setPage } = pagination;
+    const { sortOption, getSortParams } = useURLSort({
+        sortFields: ['Cluster ID'],
+        defaultSortOption: { field: 'Cluster ID', direction: 'asc' },
```

Comment:
In our Monday meeting, you mentioned creating constants for the compliance sort fields. Is that something you'll consider doing in a follow-up PR (after all these sorting tickets go in)?

Reply:
Yeah, I was wanting to get these two sorting PRs merged first and then I'll create a follow up draft for that.